### PR TITLE
feat: update copy from widgets to UI elements

### LINKED ISSUE #32331: [Task]: Code change fixes for building block on canvas
### SubTasks

- [x] Update copy in widget explorer from "Widgets" to "UI elements"
- [x] Remove highlighting of newly imported building block widgets on the canvas
- [x] Update copy from "Multi" to "Multiple selected"

### PR DESCRIPTION
**Pull request by jacquesikot**
With the addition of building blocks to the explorer items, we need to change the widget name to something that better encapsulates the contents of the canvas explorer. All instances of "widgets" in the canvas explorer have been changed to "UI elements"

Fixes #32331

## Automation

/ok-to-test tags="@tag.Widget, @tag.MainContainer, @tag.Visual"

### :mag: Cypress test results

> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/8777493414>
> Commit: 97f5cdb47a40086bf2979661c39fa4a32d0e4dd8
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=8777493414&attempt=1" target="_blank">Click here!</a>

## Summary by CodeRabbit

- **New Features**
	- Added new constants and updated text descriptions for UI elements in the editor pane.
	- Enhanced the multi-select property pane with clearer text.
- **Refactor**
	- Changed terminology from "widgets" to "UI elements" across various UI components.
- **Style**
	- Updated messages and placeholder texts to improve user interface clarity and consistency.

### CODE REVIEW

**Review by sbalaji1192 — COMMENTED** [review]
change looks good. Make sure to run all the cypress cases once.

**Review by sbalaji1192 — DISMISSED** [review]

File: app/client/src/pages/Editor/MultiSelectPropertyPane/index.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ function MultiSelectPropertyPane() {
     <div className="relative space-y-3">
       <div className="px-3 py-3">
         <Text kind="heading-s" renderAs="h3">
-          Multi
+          Multiple selected
```
Comment (mohanarpit) [inline app/client/src/pages/Editor/MultiSelectPropertyPane/index.tsx:23]: @jacquesikot As part of this change, any reason we aren't moving this to `messages.ts` as well? Ensures that all our strings are in a single location & can be easily reviewed/modified. 

I'm requesting this change as part of the "Boy Scout" rule of leaving the code a little better than we found it.

File: app/client/src/pages/Editor/widgetSidebar/UIEntitySidebar.tsx [hunk 2]
```diff
@@ -102,7 +102,7 @@ function UIEntitySidebar({
           autoComplete="off"
           id={ENTITY_EXPLORER_SEARCH_ID}
           onChange={search}
-          placeholder="Search widgets"
+          placeholder="Search UI elements"
```
Comment (rahulbarwal) [inline app/client/src/pages/Editor/widgetSidebar/UIEntitySidebar.tsx:105]: Similar to above request by Arpit, lets move this to `messages.ts`

**Review by rahulbarwal — CHANGES_REQUESTED** [review]

**Review by rahulbarwal — APPROVED** [review]

**Pull request merged**